### PR TITLE
Chore/enable run without direnv

### PR DESCRIPTION
**Pull request by kazuyaseki**
enable running without direnv using dotenv in cli.py

### CODE REVIEW

**Comment by thebongcook** [comment]
Should the `.envrc` file be removed? (Since that is connected to `direnv`)

I tried removing it locally and running `langsmith-evaluation-helper evaluate` by installing the package from this branch and it seems to fail.
Normal error 
```
langsmith.utils.LangSmithUserError: API key must be provided when using hosted LangSmith API
```

Could you please let me know if I'm missing any step?

**Comment by sandy1618** [comment]
> Shouldn't the `.envrc` file be removed? (Since that is connected to `direnv`)
> 
> I tried removing it locally and running `langsmith-evaluation-helper evaluate` by installing the package from this branch and it seems to fail. Normal error
> 
> ```
> langsmith.utils.LangSmithUserError: API key must be provided when using hosted LangSmith API
> ```
> 
> Could you please let me know if I'm missing any step?

@thebongcook 
I tried it in fresh environment and its working fine. 
image

**Comment by sandy1618** [comment]
LGTM! Thanks

**Comment by thebongcook** [comment]
> > Shouldn't the `.envrc` file be removed? (Since that is connected to `direnv`)
> > I tried removing it locally and running `langsmith-evaluation-helper evaluate` by installing the package from this branch and it seems to fail. Normal error
> > ```
> > langsmith.utils.LangSmithUserError: API key must be provided when using hosted LangSmith API
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Could you please let me know if I'm missing any step?
> 
> @thebongcook I tried it in fresh environment and its working fine. image

@sandy1618 Have you tried after deleting `.envrc`?

**Review by thebongcook — CHANGES_REQUESTED** [review]
> Shouldn't the `.envrc` file be removed? (Since that is connected to `direnv`)
> 
> I tried removing it locally and running `langsmith-evaluation-helper evaluate` by installing the package from this branch and it seems to fail. Normal error
> 
> ```
> langsmith.utils.LangSmithUserError: API key must be provided when using hosted LangSmith API
> ```
> 
> Could you please let me know if I'm missing any step?

@sandy1618 Thanks.
I double-checked and the code is OK. There was something with my env setup.

@kazuyaseki 
I think if you could just remove the `.envrc` then it would be okay to merge this PR

**Comment by thebongcook** [comment]
Post discussion memo:
After deleting `.envrc`, the package itself is running fine, but pytest seems to be failing.

**Comment by kazuyaseki** [comment]
@thebongcook 
test is working in my local after removing .envrc.
you probably neither not installing pytest-dotenv or still using direnv?
スクリーンショット 2024-07-29 11 21 19

**Comment by thebongcook** [comment]
> @thebongcook test is working in my local after removing .envrc. you probably neither not installing pytest-dotenv or still using direnv? `direnv deny` might fix the problem スクリーンショット 2024-07-29 11 21 19

@kazuyaseki 

TLDR: [This commit](https://github.com/gaudiy/langsmith-evaluation-helper/pull/17/commits/14ea49e15027f0c2e3302e35a0c95a2a224c8d42#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R280) solved the issue. `pytest-dotenv` wasn't there in the [requirements.txt of this branch last week](https://github.com/gaudiy/langsmith-evaluation-helper/blob/1887a7fd4581febb37e99e236cc4edd772d9b6ea/requirements.txt) when we checked. 

@sandy1618 Please pull the latest commits of the branch and it should work.

Details:
- `direnv deny` wasn't needed because `.envrc` no longer exists. It'll just throw an error.
```
direnv deny
direnv: error .envrc file not found
```
- Checked to see if `pytest-dotenv`  was there and it wasn't.
```
uv pip show pytest-dotenv 
warning: Package(s) not found for: pytest-dotenv
```
- git pulled and ran the test again. Worked. 
`================================================= 42 passed, 2 deselected in 8.69s =================================================`

**Review by thebongcook — APPROVED** [review]
LGTM now
https://github.com/gaudiy/langsmith-evaluation-helper/pull/17#issuecomment-2255800396
